src/modules/map/components/toolbox/TimeSlider.vue

```diff
@@ -268,6 +272,15 @@ function togglePlayYearsWithData() {
         playYearInterval = null
     }
 }
+function setYearToInputIfValid() {
+    if (
+        currentYear.value != displayedYear.value &&
+        allYears.value.includes(parseInt(displayedYear.value))
+    ) {
+        currentYear.value = parseInt(displayedYear.value)
+        setCurrentYearAndDispatchToStore(currentYear.value)
```

Comment:
currentYear is already set within 'setCurrentYearAndDispatchToStore', there is no need to assign its value here.